docs/scripts_md/run_defacing_script.md

```diff
@@ -0,0 +1,136 @@
+# NAME
+
+run\_defacing.pl -- a script that creates defaced images for anatomical
+acquisitions specified in the Config module of LORIS.
+
+# SYNOPSIS
+
+perl tools/run\_defacing\_script.pl `[options]`
```

Comment:
only options show as code, suggest to include the entire line in quotes or no quotes at all.